app/views/items/edit.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :sidebar do %>
   <h1>Editing Item</h1>
-  <%= sidebar_button_link 'Cancel Edit', :back %>
+  <%= sidebar_button_link 'Cancel', item_type_path(@item_type) %>
```

Comment:
i'd prefer `item_type_path(@item.item_type)` (and getting rid of `@item_type` in the controller.